bundles/org.openhab.core/src/main/java/org/openhab/core/items/events/ItemEventFactory.java

```diff
@@ -268,6 +281,26 @@ public static ItemEvent createStateEvent(String itemName, State state) {
         return createStateEvent(itemName, state, null);
     }
 
+    /**
+     * Creates an item historic state event.
+     *
+     * @param itemName the name of the item to send the state update for
+     * @param state the new state to send
+     * @param source the name of the source identifying the sender (can be null)
+     * @return the created item state event
+     * @throws IllegalArgumentException if itemName or state is null
+     */
+    public static ItemHistoricStateEvent createHistoricStateEvent(String itemName, State state, ZonedDateTime dateTime,
+            @Nullable String source) {
+        assertValidArguments(itemName, state, "state");
+        // TODO: use a different topic for historic state ? Would require changes to ItemUpdater.
```

Comment:
You already defined a constant for that above. Why don't you use it here? IMO the topic has to be different, because the payload is different and the event also serves a different purpose